client/client.go

```diff
@@ -69,6 +71,44 @@ type ApiRequester interface {
 	Do(req *http.Request) (*http.Response, error)
 }
 
+// APIError wraps an error with the request_id - and if
```

Comment:
`... and is also an error type returned after receiving an error message from the server`.

Reply:
agree